packages/icons/src/library/link-off.js

```diff
@@ -5,7 +5,7 @@ import { SVG, Path } from '@wordpress/primitives';
 
 const linkOff = (
 	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-		<Path d="M15.6 7.3h-.7l1.6-3.5-.9-.4-3.9 8.5H9v1.5h2l-1.3 2.8H8.4c-2 0-3.7-1.7-3.7-3.7s1.7-3.7 3.7-3.7H10V7.3H8.4c-2.9 0-5.2 2.3-5.2 5.2 0 2.9 2.3 5.2 5.2 5.2H9l-1.4 3.2.9.4 5.7-12.5h1.4c2 0 3.7 1.7 3.7 3.7s-1.7 3.7-3.7 3.7H14v1.5h1.6c2.9 0 5.2-2.3 5.2-5.2 0-2.9-2.4-5.2-5.2-5.2z" />
+		<Path d="M17.0312 4.70312L15.5756 4L14.0156 7.00022H14V7.03028L11.6759 11.5H9.5V13H10.896L9.39388 15.8889H8.44445C6.40406 15.8889 4.75 14.2348 4.75 12.1944C4.75 10.1541 6.40406 8.5 8.44445 8.5H10V7H8.44445C5.57563 7 3.25 9.32563 3.25 12.1944C3.25 15.0633 5.57564 17.3889 8.44445 17.3889H8.61394L7.5 19.5312L8.91562 20.25L15.0491 8.50022H15.5555C17.5959 8.50022 19.2499 10.1543 19.2499 12.1947C19.2499 14.235 17.5959 15.8891 15.5555 15.8891H14V17.3891H15.5555C18.4243 17.3891 20.7499 15.0635 20.7499 12.1947C20.7499 9.41742 18.5704 7.14923 15.8285 7.00727L17.0312 4.70312Z" />
```

Comment:
Did you run this one through SVGO? It's nitpick territory, but it looks like the values can be rounded up a little bit to 2 decimals rather than 4.

I like using https://svgomg.net/.

Reply:
Thanks! It rounded them a little more.

Reply:
I used the Figma SVG Export plugin.